deltachat-ios/View/TextFieldCell.swift

```diff
@@ -83,13 +137,39 @@ class TextFieldCell: UITableViewCell {
         textField.text = text
     }
 
+    override func traitCollectionDidChange(_ previousTraitCollection: UITraitCollection?) {
+        if previousTraitCollection?.preferredContentSizeCategory !=
+            traitCollection.preferredContentSizeCategory {
+            updateViews()
+        }
+    }
+
+    override func prepareForReuse() {
+        super.prepareForReuse()
+        title.text = nil
+        title.attributedText = nil
+        textField.text = nil
+    }
+
+    private func updateViews() {
+        if fontSize <= maxFontSizeHorizontalLayout {
```

Comment:
for large font sizes the text field is below the text title in order to have enough space to read both and/or write into the text field